lib/wings/services/file_converter_service.rb

```diff
@@ -0,0 +1,66 @@
+# Conversion service for going between files on a valkyrie resource and files on an active fedora object
+module Wings
+  class FileConverterService
```

Comment:
where do we plan to call this from?

Reply:
These methods will be called from ModelTransformer and ActiveFedoraConverter.  That is what I'm working on next.  In master now, this conversion of the file metadata is not happening.